dbld/images/required-epel/all.txt

```diff
@@ -2,3 +2,5 @@
 libdbi
 libdbi-drivers
 riemann-c-client-devel
+syslog-ng-java-deps
```

Comment:
Where does this package come from?

Reply:
From my Copr repo: https://copr.fedorainfracloud.org/coprs/czanik/syslog-ng-githead/package/syslog-ng-java-deps/

Reply:
I think it's not the githead branch in your repo, but version "3.11":
https://github.com/balabit/syslog-ng/blob/4ab34df81a7f97740e60aa5b0748c54b6518338b/dbld/images/helpers/functions.sh#L14-L17
This is used during docker image creation.